model/src/w3wavemd.F90

```diff
@@ -1453,6 +1453,12 @@ SUBROUTINE W3WAVE ( IMOD, ODAT, TEND, STAMP, NO_OUT &
         call print_memcheck(memunit, 'memcheck_____:'//' WW3_WAVE TIME LOOP 13')
         !
 #ifdef W3_PDLIB
+        !WRITE(*,*) 'B_JGS_LDIFR', B_JGS_LDIFR
```

Comment:
Should the commented out debugging code go ahead and be deleted before merging into the code?